docs/guide/tutorials.rst

```diff
@@ -0,0 +1,431 @@
+	then select the injections and transitions to plot from their respective tabs on the left. The following shows the chromatogram
+	for two injections using Antranilicacid and 5-HTP2 transitions and their intensity differences over time.
+
+	.. image:: ../images/hplc_uv_unknowns_chromatogram.png
+
+#. Running the workflow in SmartPeak
+
+	To run the analysis, please follow the steps for 
+	:ref:`Using SmartPeak GUI` or :ref:`Using SmartPeak CLI`
+	to execute the workflow steps and review the results including plotting.
+
+#. Reporting the results
+
+	To export the results, select "Report" from the "Actions" which will show the 
+	"Create Report" window:
+
+	.. image:: ../images/hplc_uv_standards_exports.png
+
+	Based in the data you wish to export, select the desired "Sample types" from the left pane
+	and select the "Metadata" from the right pane then click on of the buttons below to create
+	the report with the selected items in the csv format. More details on exporting the results can be found 
+	in :ref:`Export report`.
+
+
+
+
+Targeted quantitation with LC-MS/MS 5500 QTRAP RapidRIP
+-------------------------------------------------------
+
+
+
+Targeted flux analysis with LC-MS/MS 5500 QTRAP
+-----------------------------------------------
+
+
+
+Targeted flux analysis with GC-MS full-scan Agilent
+---------------------------------------------------
+
+
+
+Targeted flux analysis with GC-MS SIM Agilent
+---------------------------------------------
+
+
+
+Non-targeted FIA-MS analysis with Thermo Orbitrap
+-------------------------------------------------
+
+This tutorial walks you through the workflow for analyzing targeted HPLC data
+starting from input file generation, to processing the data in SmartPeak, 
+to reviewing the data in SmartPeak, to reporting the results for later use.
+
+Objectives
+~~~~~~~~~~
+
+#. Obtaining the SOP for the workflow.
+#. Choosing a data set for demonstrating the workflow.
+#. Creating an optimized SmartPeak input templates for running the workflow.
+
+The Workflows include
+~~~~~~~~~~~~~~~~~~~~~
+
+#. Calculating the calibration curves using Standards
+#. Processing Unknowns
+
+Steps
+~~~~~
+
+The tutorial includes the following steps :
+
+#. Setting up the input files
+
+The data set used can be found here 
+`FIAMS FullScan Unknowns <https://github.com/AutoFlowResearch/SmartPeak/tree/develop/src/examples/data/FIAMS_FullScan_Unknowns>`_.
+
+#. Defining the workflow in SmartPeak
+
+For FIAMS FullScan Unknowns analysis, the following steps are saved 
+into the ``workflow.csv`` file. Alternatively, steps can be replaced, 
+added or deleted direclty from SmartPeakGUI within the "workflow" tap in the right pane. 
+A detailed explanation of each command step
+can be found in :ref:`Workflow Commands`.
+
+	* LOAD_RAW_DATA
+	* EXTRACT_SPECTRA_WINDOWS
+	* MERGE_SPECTRA
+	* PICK_MS1_FEATURES
+	* SEARCH_ACCURATE_MASS
+	* STORE_ANNOTATIONS
+	* STORE_FEATURES
+	* ESTIMATE_FEATURE_BACKGROUND_INTERFERENCES
```

Comment:
These steps would benefit from an explanation.  It would also be a good opportunity to discuss the input files used for the background interference as well as the threshold chosen for removing background features based on the "Blank" samples.

Reply:
The threshold for removing blank features is not to be found in the parameters.csv file. was it supposed to be included there?

Reply:
I am not sure there a file with that name in the FIA-MS dataset folder : https://github.com/AutoFlowResearch/SmartPeak/tree/develop/src/examples/data/FIAMS_FullScan_Unknowns.